userspace/libscap/CMakeLists.txt

```diff
@@ -202,6 +202,10 @@ if(CMAKE_SYSTEM_NAME MATCHES "Linux")
 	add_subdirectory(engine/kmod)
 	target_link_libraries(scap scap_engine_kmod)
 
+	add_definitions(-DHAS_ENGINE_TEST_INPUT)
```

Comment:
Because this is specifically meant for testing, think we can hide its compilation behind a configuration option like `CREATE_TEST_TARGETS`? Particularly, I think builds meant for release/production should not include this engine.

Reply:
I agree, also cmakefiles will need to be modified to remove this if not requested.